tests/plugins/emoji/basic.js

```diff
@@ -210,7 +225,19 @@
 				} );
 
 				arrayAssert.itemsAreSame( expected, actual );
-				assert.areSame( '😻 :smiling_cat_face_with_heart-eyes:', autocomplete.view.element.getChild( 0 ).getText(), 'First element in view should start from "smiling".' );
+				assert.areSame( '😻 smiling_cat_face_with_heart-eyes' , autocomplete.view.element.getChild( 0 ).getText(), 'First element in view should start from "smiling".' );
```

Comment:
With this PR the emoji symbol is wrapped in `<span>`, maybe in such case, we could compare inner Html instead of raw text? I feel it could be a safer approach here? WDYT?

And the same situation appears in line 240